publications/publishing-protocols.md

```diff
@@ -0,0 +1,57 @@
+# Publishing Protocols for Project Deliverables
+
+This guide helps TAG Security project leads deliver their artifacts effectively by leveraging past experiences and best practices.
+
+For detailed guidelines on creating papers and the expected standard, refer to [Process for Creating Papers](paper-process.md) and [Publishing Guidelines and Standards](authoring-guidelines.md).
+
+## Using GitHub Issues to Track Progress
+
+All deliverables need a central place for:
+
+- Expressing interest in collaboration
+- Finding the current status and timelines
+- Viewing remaining steps and tasks
+
+A GitHub Issue serves well for this purpose as it allows continuous updates without changing the location.
+
+## Identifying External Dependencies
+
+Recognize external dependencies such as CNCF Service Desk, design teams, and TOC reviewers. Start earlier than necessary to accommodate their timelines and potential delays. Show grace during unforeseen delays and reassess deadlines and scope when needed.
+
+## Get Help from CNCF Service Desk
+
+CNCF can assist with:
+
+- Professionally designed PDFs
+- Animations, graphics, and diagrams
+- Blog posts
+- Social media outreach
+
+Publishing around KubeCon allows for broader sharing through various forums and sessions.
+
+## Define Stages of Publication
+
+Treat the process like a project with milestones. Identify milestone leads for large projects. A tentative schedule for a paper publication process can help track progress.
+
+## Write a Blog
+
+A blog post acts as a trailer for the deliverable. It should include:
+
+- A brief summary
+- Link to the deliverable
+- Quotes from contributors or project lead
+- Reasons why the readers should care
+
+## Common Mistakes to Avoid
+
+- Review the artifact for brevity, correctness, and flow multiple times with different goals.
+- Ensure all contributors and reviewers are credited in the PDF or markdown.
+- Engage CNCF early to avoid last-minute rushes.
```

Comment:
I'd add "Define the scope early in the process, and revisit it often to avoid scope creep"